### PR TITLE
New Package: Dwarf-Therapist-41.1.7

### PR DESCRIPTION
**Pull request by Eluminae**
A updated version of https://github.com/void-linux/void-packages/pull/10800

* Adding ptrace utils provided by therapist

### CODE REVIEW

**Comment by abenson** [comment]
Package name should Dwarf-Therapist, as that's what upstream uses

**Comment by Eluminae** [comment]
> Package name should be Dwarf-Therapist, as that's what upstream uses

done !

**Comment by abenson** [comment]
Can you update this 41.2.0?

**Comment by ericonr** [comment]
@Eluminae ping

**Comment by Eluminae** [comment]
done

File: srcpkgs/Dwarf-Therapist/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+short_desc="Management tool designed to run side-by-side with Dwarf Fortress"
+maintainer="Robert Stancil <robert.stancil@mavs.uta.edu>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/Dwarf-Therapist/Dwarf-Therapist/"
+distfiles="https://github.com/Dwarf-Therapist/Dwarf-Therapist/archive/v${version}.tar.gz"
+checksum=ec954daff6e03d9d44153ed3df90685de3a7a07431fd47639d83b2f5b3f62e4b
+
+post_install() {
+	vbin dist/ptrace_scope/patch_df_ptracer
+	vbin dist/ptrace_scope/ptrace_cap_wrapper
```
Comment (ericonr) [inline srcpkgs/Dwarf-Therapist/template:17]: Is this wrapper supposed to be suid / have capabilities? Afaik those won't be kept when installed with `vbin`

Comment (Eluminae) [reply srcpkgs/Dwarf-Therapist/template:17]: I dont think they need any specific suid. To be honest, this mr has been created a long time ago.

But I'm pretty sure I used this package ootb

**Pull request merged**